### DISCUSSION TITLE
how to use exceljs in angular 7?

### DISCUSSION THREAD

**Discussion by ankitsalu** [exceljs/exceljs#2494] [Q&A]
Is there a good way rather than doing below:
import * as Excel from "exceljs/dist/exceljs.min.js";

If I don't do the above then i get errors like:
Module not found: Error: Can't resolve 'fs'.....stream, zip etc

**Comment by Senneseph** [comment]
In my tsconfig.json, I have this:

>{
  "compileOnSave": false,
  "compilerOptions": {
    "paths": {
      "exceljs": ["node_modules/exceljs/dist/exceljs.min"],
    },
  ...

And then in my component, I can do: `import { Workbook } from 'exceljs'`

However, the whole exceljs library gets pulled in, even if your are only using a single export.  I'd like to know if there is a way to slim it down and if it requires some code changes / cleanup I think I have the time to contribute.
